sync/class.jetpack-sync-listener.php

```diff
@@ -173,6 +173,15 @@ function enqueue_action( $current_filter, $args, $queue ) {
 			return;
 		}
 
+		/**
+		 * Add an action hook to execute when anything on the whitelist gets sent to the queue to sync.
+		 *
+		 * @since 5.8
+		 *
+		 * @param array The action parameters
+		 */
+		do_action( "jetpack_action_before_enqueue" );
```

Comment:
I think you could use single quotes here. And I would suggest renaming the action to `jetpack_sync_action_before_enqueue`, so it's a bit more descriptive.